### PR TITLE
Remove virtual warehouses

### PR DESCRIPTION
**Pull request by k-sharpe**

### CODE REVIEW

File: postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java [hunk 1]
```diff
@@ -34,13 +54,20 @@
...
         (String) data.getProperty("warehouseLatitude"),
         (String) data.getProperty("warehouseLongitude"),
         (String) data.getProperty("customerLatitude"),
         (String) data.getProperty("customerLongitude"),
         (String) data.getProperty("shipWeightValue"), (String) data.getProperty("distance"),
-        (String) data.getProperty("warehouseId"), (String) data.getProperty("shippingLevel"));
+        updatedWarehouseId, (String) data.getProperty("shippingLevel"));
+  }
+
+  private static String convertVirtualToRealWarehouse(String warehouseId) {
```
Comment (jgao23) [inline postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java:70]: supernit: there should be a default if the warehouseId doesn't exist in warehouseIdToRealId

Comment (jgao23) [reply postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java:70]: are there tests for this that should be updated?

Comment (jennygaoo) [reply postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java:70]: +1

Comment (k-sharpe) [reply postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java:70]: No tests, going to workon that in the more testing branch

File: postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java [hunk 2]
```diff
@@ -7,17 +7,37 @@
...
+import java.util.Map;
 /**
  * This class is used to read order information from a datastore service to create java order
  * objects.
  */
 public class OrderProcessor {
   private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+  // Part of the conversion for virtual warehouses into shared real warehouse for shipping purposes.
+  private static final Map<String, String> warehouseIdToRealId =
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/common/OrderProcessor.java:23]: WAREHOUSE_ID_TO_REAL_ID

**Pull request merged**